src/styles/typography/index.md.njk

```diff
@@ -19,13 +19,11 @@ If you’re not sure whether you should be using GDS Transport, read the Service
 
 ## Headings
 
-Use headings consistently to create a clear hierarchy throughout your service.
+Mark up headings semantically using the appropriate <h#> level HTML element and use a heading class to apply the GOV.UK styling. 
 
-Mark up headings semantically using the appropriate <h#> level HTML element and use a heading class to apply the GOV.UK styling.
+The styling should reflect the hierarchy of the page. If your `h1` is styled as `govuk-heading-l`, your `h2`s should be styled as `govuk-heading-m` and so on.
```

Comment:
This revision removes the nuance from the previous content that means it now strongly promotes 1:1 relationship between the presentational class and the semantic heading level.

It's a good thing for example to use a bigger visual (presentational) heading with a smaller semantic level in certain contexts, otherwise you can encourage incorrect heading usage as people think they need a h1 to have a big heading.

Reply:
Good point, that was definitely not my intention. Have revised the guidance to consider this